A3-channel-tracing.md

```diff
@@ -53,27 +53,25 @@ The data will be accessed via the channelz service, which sends a ChannelTrace p
 // the definitions of these protos can be found in A14-channelz.md
 message ChannelRef {}
 message SubchannelRef {}
-message ChannelConnectivityState {}
 
 // A trace event is an interesting thing that happened to a channel or
 // subchannel, such as creation, address resolution, subchannel creation, etc.
 message ChannelTraceEvent {
   // High level description of the event.
   string description = 1;
-  // Status/error associated with this event.
-  // Optional, only present is Status != OK. 
-  google.rpc.Status status = 2;
+  // the severity of the trace event. Options are INFO, WARNING, and ERROR,
```

Comment:
With regard to log levels, I agree that DEBUG is not useful, because we're not going to want to waste memory for anything that is more detailed than what we're already recording with INFO.  I also suspect that WARNING is not going to get much use, because in practice most messages are either INFO or ERROR, but I'm not necessarily opposed to including it.  So I think the two main ones we need are INFO and ERROR, which is why I suggested that this be a bool, but I'm not opposed to also supporting WARNING and making this an enum instead.

In terms of the type of this field, we should either decide to support only two types and make this a bool, or we should make it an enum.  I don't think it makes sense for the field to be an int with documentation describing enum semantics; if we really want an enum, let's use one.

Reply:
I have made severity into an enum. I'd like to support WARNING, I think it would be useful for events that still occur in healthy channels, but need a closer look, because a lot of them might signal an underlying issue.